### PR TITLE
Fix batch update edge cases for AddCommand and DeleteCommand

### PR DESCRIPTION
**Pull request by whoisjunhong**

### CODE REVIEW

File: docs/team/whoisjunhong.md [hunk 1]
```diff
@@ -23,18 +23,30 @@ Given below are my contributions to the project.
...
   * Credits: *{Collaborators: Jun Hong, Joshua, Louis}*
 
+* **Fixes**: Fix batch update status for `AddCommand` and `DeleteCommand`  -- (Pull request: [\#118](https://github.com/AY2122S2-CS2103T-T12-1/tp/pull/118))
+* What it fixes:
+* Edge Case 1: If user is added into application who is `Positive`, the students in the same ClassCode and Activity does not change to `Close-Contact`
+* Edge Case 2: If user is deleted from the application who is `Positive` and the ClassCode and/or Activity does not have any other positive cases, those students statuses does not change to `Negative`
+* Edge Case 3: If user is added into application who is `Negative` while the ClassCode and/or Activity is `Close-Contact`, those students statuses does not change to `Negative`
+* Edge Case 4: If user is added into application who is `Negative` while there is still a `Positive` case in the ClassCode and/or Activity, the recently added student status will be changed to `Close-Contact`
+* Justification: Fix edge cases which will snowball into other critical bugs in the future
+* Credits: *Jun Hong*
```
Comment (lzf834) [inline docs/team/whoisjunhong.md:33]: Good catch on the bugs as discussed ystdy.

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 2]
```diff
@@ -60,10 +76,96 @@ public CommandResult execute(Model model) throws CommandException {
...
+                    EditCommand.EditPersonDescriptor tempDescriptor = new EditCommand.EditPersonDescriptor();
+                    tempDescriptor.setStatus(new Status(Status.NEGATIVE));
+                    Person editedPersonStatus = createEditedPerson(currentPerson, tempDescriptor);
+                    model.setPerson(currentPerson, editedPersonStatus);
+                } else {
+                    EditCommand.EditPersonDescriptor tempDescriptor = new EditCommand.EditPersonDescriptor();
+                    tempDescriptor.setStatus(new Status(Status.CLOSE_CONTACT));
+                    Person editedPersonStatus = createEditedPerson(addedPerson, tempDescriptor);
+                    model.setPerson(addedPerson, editedPersonStatus);
+                }
```
Comment (lzf834) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:144]: code looks okay albeit a little long. Can leave it like that. but I think if we are to improve on the app further might need to abstract this out

**Review by lzf834 — APPROVED** [review]
Good catch. code looks good

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 2 above)
Comment (whoisjunhong) [reply src/main/java/seedu/address/logic/commands/AddCommand.java:144]: Agreed, we can disucss this on v1.4 and see what we can do about this 😄

**Pull request merged**